src/core/src/opsets/opset.cpp

```diff
@@ -207,3 +227,23 @@ const ngraph::OpSet& ngraph::get_opset10() {
     static OpSet opset(ov::get_opset10());
     return opset;
 }
+
+#define _NGRAPH_REG_OPSET(OPSET) opset_map[#OPSET] = ngraph::get_##OPSET;
+
+const std::map<std::string, std::function<const ngraph::OpSet&()>>& ngraph::get_available_opsets() {
+    static std::map<std::string, std::function<const ngraph::OpSet&()>> opset_map;
+    static std::once_flag flag;
+    std::call_once(flag, [&]() {
+        _NGRAPH_REG_OPSET(opset1);
```

Comment:
1. Why do we need ngraph and ov function? can we have a single one?
2. Why do we need once_flag? can we just create map as const object and perform assignment during construction?
3. Can we add its usage in ie_core.cpp as well?

Reply:
We have different opset objects for `ngraph` and `ov`.
All comments fixed.